packages/calcite-components/src/utils/resources.ts

```diff
@@ -41,3 +41,10 @@ export const CSS_UTILITY = {
 export const TEXT = {
   loading: "Loading",
 };
+
+export const debounceValues = {
+  filter: 250,
+  nextTick: 0,
+  overflow: 150,
```

Comment:
I wonder if we should rename `overflow` to `resize`?

Reply:
Yeah, in this case it's wrapping the resize method.

Reply:
if its wrapping the resize method, maybe it can just use the resize value?

Reply:
Sounds good. Lets rename to `resize` 👍

Reply:
I dig it!